watchtower/wtclient/manager.go

```diff
@@ -21,9 +21,9 @@ import (
 	"github.com/lightningnetwork/lnd/watchtower/wtpolicy"
 )
 
-// TowerClientManager is the primary interface used by the daemon to control a
+// ClientManager is the primary interface used by the daemon to control a
 // client's lifecycle and backup revoked states.
-type TowerClientManager interface {
+type ClientManager interface {
```

Comment:
Since the package is called `wtclient`, we could even just call this `Manager`. But non-blocking of course.

Reply:
the reason for leaving this as `ClientManager` is cause `Manager` is already used below for the name of the impl struct